app/src/pages/settings/SettingsAnnotationsPage.tsx

```diff
@@ -0,0 +1,248 @@
+          onError: parseError(onError),
+        });
+        break;
+    }
+    revalidate();
+  };
+
+  const [updateContinuousAnnotationConfig] = useMutation(graphql`
+    mutation SettingsAnnotationsPageUpdateContinuousAnnotationConfigMutation(
+      $input: UpdateContinuousAnnotationConfigInput!
+    ) {
+      updateContinuousAnnotationConfig(input: $input) {
+        annotationConfig {
+          id
+        }
+      }
+    }
+  `);
+
+  const [updateCategoricalAnnotationConfig] = useMutation(graphql`
+    mutation SettingsAnnotationsPageUpdateCategoricalAnnotationConfigMutation(
+      $input: UpdateCategoricalAnnotationConfigInput!
+    ) {
+      updateCategoricalAnnotationConfig(input: $input) {
+        annotationConfig {
+          id
+        }
+      }
+    }
+  `);
+
+  const [updateFreeformAnnotationConfig] = useMutation(graphql`
+    mutation SettingsAnnotationsPageUpdateFreeformAnnotationConfigMutation(
+      $input: UpdateFreeformAnnotationConfigInput!
+    ) {
+      updateFreeformAnnotationConfig(input: $input) {
+        annotationConfig {
+          id
+        }
+      }
+    }
+  `);
+
+  const handleEditAnnotationConfig = (
+    _config: AnnotationConfig,
+    {
+      onCompleted,
+      onError,
+    }: { onCompleted?: () => void; onError?: (error: string) => void } = {}
+  ) => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { id, annotationType, ...config } = _config;
+    switch (annotationType) {
+      case "CONTINUOUS":
+        updateContinuousAnnotationConfig({
+          variables: {
+            input: {
+              ...config,
+              configId: id,
+            },
+          },
+          onCompleted,
+          onError: parseError(onError),
+        });
+        break;
+      case "CATEGORICAL":
+        updateCategoricalAnnotationConfig({
+          variables: {
+            input: {
+              ...config,
+              configId: id,
+            },
+          },
+          onCompleted,
+          onError: parseError(onError),
+        });
+        break;
+      case "FREEFORM":
+        updateFreeformAnnotationConfig({
+          variables: {
+            input: {
+              ...config,
+              configId: id,
+            },
+          },
+          onCompleted,
+          onError: parseError(onError),
+        });
+        break;
+    }
+    revalidate();
+  };
+
+  const handleDeleteAnnotationConfig = (
+    id: string,
+    {
+      onCompleted,
+      onError,
+    }: { onCompleted?: () => void; onError?: (error: string) => void } = {}
+  ) => {
+    deleteAnnotationConfig({
+      variables: { input: { configId: id } },
+      onCompleted,
+      onError: parseError(onError),
+    });
+    revalidate();
+  };
+
+  return (
+    <Card
+      title="Annotation Configs"
+      variant="compact"
+      bodyStyle={{ padding: 0 }}
+      extra={
+        <DialogTrigger>
+          <Button size="S">New Annotation Config</Button>
```

Comment:
Maybe a + icon with the word "New Configuration" - just to keep things more proper.